src/constants/makeApplyText.ts

```diff
@@ -0,0 +1,7 @@
+export const makeApplyKustomizationText = (name: string, context: string) =>
```

Comment:
Let's chabge the context type to `string | undefined` and use it as `${context || ''}` below so we don't have to do it where the function is being used